helpers/list-dependencies.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+import os
+
+for dir in os.listdir("/nix/store"):
```

Comment:
I don't know what this is used for, but in general listing the nix store is such a bad idea that [it will be forbidden in the future](https://github.com/NixOS/rfcs/pull/97). Besides, one should not hard-code `/nix/store` too much as that path technically is configurable via environment variables.

Reply:
Thank you for the feedback.

I did not know it would be forbidden, but nice to know.
Though, I'm not sure this will ever be a problem as the code here runs inside a Docker container, not on NixOS.

This approach was the first working option I've found to extract the requirements and scan them for vulnerability (nice tab in the build [logs](https://github.com/LeastAuthority/magic-wormhole-docker/actions/runs/4974686756)).

As a possible improvement for later, I'm thinking to achieve the same with an extra derivation (with `copyToRoot` from [`dockerTools/buildLayeredImage`](https://ryantm.github.io/nixpkgs/builders/images/dockertools/) maybe).
But this will cost me too much time.
It is only to scan the Python modules and this whole part could be remove/replace if not stable.

Reply:
I've pushed d0fe09a18d17253dd07f45aca5b67ba6dc722965 to use the environment variable in the new script.
But this will unlikely be complete unless the variable is also define at higher level (GitHub Runner/Action!?).